nengo/tests/test_copy.py

```diff
@@ -11,15 +10,6 @@
 from nengo.utils.testing import warns
 
 
-@pytest.fixture(scope='module', autouse=True)
-def error_on_not_added_to_network_warning(request):
-    catcher = warnings.catch_warnings(record=True)
-    catcher.__enter__()
-    warnings.simplefilter(
-        'error', category=NotAddedToNetworkWarning)
-    request.addfinalizer(lambda: catcher.__exit__(None, None, None))
```

Comment:
It makes all the `with warns(NotAddedToNetworkWarning)` tests fail (since they error when the warning is raised). It seemed more complicated than it was worth to disable this check in the cases in which we wanted to check the warning. Any ideas on how to do it simply?